62.md

```diff
@@ -0,0 +1,112 @@
+NIP-62
+======
+
+Curated Publication
+----------
+
+`draft` `optional`
+
+This NIP defines the minimum specification for curated publication - ordered, optionally-hierarchical assemblies of nostr events. Collections provide a standard way to organize and present related content, similar to how books organize chapters or journals organize articles.
```

Comment:
Typos:
* "curated publication" should be "curated publications
* "nostr" should be "Nostr"